elyra/metadata/manager.py

```diff
@@ -84,8 +84,8 @@ def get_all(self, include_invalid: bool = False, of_schema: str = None) -> List[
                 # Since we may not have a metadata instance due to a failure during `from_dict()`,
                 # instantiate a bad instance directly to use in the message and invalid result.
                 invalid_instance = Metadata(**metadata_dict)
-                self.log.debug("Fetch of instance '{}' of schemaspace '{}' encountered an exception: {}".
-                               format(invalid_instance.name, self.schemaspace, ex))
+                self.log.warning("Fetch of instance '{}' of schemaspace '{}' encountered an exception: {}".
+                                 format(invalid_instance.name, self.schemaspace, ex))
```

Comment:
Would you mind converting this to an f-string? We're trying to make those kinds of changes as we go along.
```suggestion
                self.log.warning("Fetch of instance '{invalid_instance.name}' of schemaspace '{self.schemaspace}' encountered an exception: {ex}".)
```
The result will likely need to be split, but this improves readability.

Reply:
Oops, just realized I forgot the 'f'! `self.log.warning(f"Fetch ...`. So much for it being an f-string!